src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -65,26 +72,60 @@
     private final transient String key;
 
     /**
+     * Request for fetching app token.
+     */
+    private final transient com.jcabi.http.Request trequest;
+
+    /**
+     * Request for verifying user credentials.
+     */
+    private final transient com.jcabi.http.Request vcrequest;
```

Comment:
@pinaf `token` is not possible due to:
[ERROR] \src\main\java\org\takes\facets\auth\social\PsTwitter.java[76-76]: Field token has the same name as a method (AvoidFieldNameMatchingMethodName)